src/models/__tests__/civilUnion.test.js

```diff
@@ -147,6 +147,17 @@ describe('The civilUnion model', () => {
       .toEqual(expect.arrayContaining(expectedErrors))
   })
 
+  it('the civilUnion telephone field is required', () => {
+    const testData = {
+      Telephone: '1234567890',
```

Comment:
can this test against `{ noNumber: true }` like the other ones in this PR?

Reply:
Yes! Updated